src/analysis/utils/cost-utils.ts

```diff
@@ -282,9 +280,7 @@ export const getPersistentDiskCostMonthly = (disk: DecoratedPersistentDisk, comp
 };
 
 export const getPersistentDiskCostHourly = (
-  disk:
-    | DecoratedPersistentDisk
-    | { size: number; status: LeoDiskStatus; diskType: GooglePdType; cloudContext?: CloudContext },
+  disk: PersistentDisk | { size: number; status: LeoDiskStatus; diskType: GooglePdType; cloudContext?: CloudContext },
```

Comment:
nit: if you only need these 4 fields off the PD type, you could do `disk: Pick<PersistentDisk, ['size', 'status', 'diskType', 'cloudContext']>`